### PR TITLE
WindowsHostNetwork alpha updates

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

### CODE REVIEW

**Comment by marosset** [comment]
Part of https://github.com/kubernetes/enhancements/issues/3503

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ different Kubernetes components.
 | `WinOverlay` | `true` | Beta | 1.20 | |
 | `WindowsHostProcessContainers` | `false` | Alpha | 1.22 | 1.22 |
 | `WindowsHostProcessContainers` | `true` | Beta | 1.23 | |
+| `WindowsHostNetwork` | `false` | Alpha | 1.26| |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:200]: move this before WindowsHostProcessContainers ?

File: content/en/docs/concepts/windows/intro.md [hunk 2]
```diff
@@ -261,6 +261,17 @@ The following list documents differences between how Pod specifications work bet
...
 
+#### Field compatibility for hostNetwork {#compatibility-v1-pod-spec-containers-hostnetwork}
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+The kubelet can now request that pods running on Windows nodes use the host's network namespace instead
+of creating a new pod network namespace. To enable this functionality pass `--feature-gates=WindowsHostNetwork=true` to the kubelet.
+
+{{< note >}}
+This functionality requires a container runtime that also supports this functionality.
```
Comment (tengqm) [inline content/en/docs/concepts/windows/intro.md:272]: remove "also"?

**Comment by krol3** [comment]
Hi @marosset! This PR needs a doc review by **Mon Nov 28th** to get this into the release. Please reach out to required SIGs to get their review. Thank you!

**Pull request merged**